src/ethereum_spec_tools/evm_tools/loaders/fork_loader.py

```diff
@@ -40,12 +40,17 @@ def proof_of_stake(self) -> bool:
         raise Exception(f"fork {self._fork_module} not discovered")
 
     def is_after_fork(self, target_fork_name: str) -> bool:
-        """Check if the fork is after the target fork."""
+        """
+        Check if the fork is after the target fork.
+
+        Accepts short fork names (e.g. "cancun") instead of full module paths.
+        """
         return_value = False
         for fork in self._forks:
-            if fork.name == target_fork_name:
+            short_name = fork.name.split(".")[-1]
```

Comment:
You should use [`short_name`](https://github.com/ethereum/execution-specs/blob/26c7daf42c968e0715fc69dbe6876c2a6a1601dd/src/ethereum_spec_tools/forks.py#L240)